packages/near-membrane-base/src/membrane.ts

```diff
@@ -1062,13 +1199,27 @@ export function createMembraneMarshall() {
                 key: string | symbol,
                 receiver: any
             ): ReturnType<typeof Reflect.get> {
-                return getLocalValue(
-                    foreignCallableGet(
-                        this.foreignTargetPointer,
-                        key,
-                        getTransferableValue(receiver)
-                    )
+                const { foreignTargetPointer, foreignTargetTraits } = this;
+                const result = getLocalValue(
+                    foreignCallableGet(foreignTargetPointer, key, getTransferableValue(receiver))
                 );
+                if (result === undefined) {
+                    if (
+                        key === LOCKER_NEAR_MEMBRANE_SYMBOL &&
+                        !foreignCallableHas(foreignTargetPointer, key)
+                    ) {
+                        return true;
+                    }
+                    const handlerKey = PROPERTY_TO_HANDLER_KEY_MAP[key];
+                    if (
+                        handlerKey !== undefined &&
+                        foreignTargetTraits & TargetTraits.IsObject &&
+                        !foreignCallableHas(foreignTargetPointer, key)
+                    ) {
+                        return this[handlerKey];
```

Comment:
this is specially weird for me.

Reply:
We are only concerned with potentially getting the value of `this.unbrandedTag` or `this.serializedValue` if the value is not-an array or function since the value of both would be `undefined` and the additional `foreignCallableHas` would be wasted on an `undefined` result. We want to ensure the symbol keys are not actually present on these objects and having an explicit `undefined` value so we check with `foreignCallableHas`. In other words we want our symbols magic and not to be bolted on to user-land objects and we want to do these checks if we are more sure there is actually going to be a value generated from the `this.unbrandedTag` and `this.serializedValue` deferred getter.